src/api/todos.ts

```diff
@@ -0,0 +1,19 @@
+import { Todo } from '../types/Todo';
+import { NewTodo } from '../types/NewTodo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const addTodos = (userId: number, data: NewTodo) => {
+  return client.post<Todo[]>(`/todos?userId=${userId}`, data);
```

Comment:
The return type should be 'Todo' instead of 'Todo[]' because the function adds a single todo.